lib/src/clipboard/clipboard.dart

```diff
@@ -0,0 +1,16 @@
+
+import 'package:flutter/services.dart';
+import 'package:flushbar/flushbar.dart';
+
+final flushbarCopied = Flushbar<dynamic>(
+  message: 'Message copied to clipboard',
+  flushbarPosition: FlushbarPosition.BOTTOM,
+  flushbarStyle: FlushbarStyle.FLOATING,
+  borderRadius: 20,
+  isDismissible: true,
+  duration: Duration(milliseconds: 20),);
+
+Flushbar copyToClipboard(String textToCopy) {
```

Comment:
As said previously, take the context in the function as a first argument and call `show()` in the function